pkg/ingester/ingester.go

```diff
@@ -273,9 +273,10 @@ func NewForFlusher(cfg Config, chunkStore ChunkStore, registerer prometheus.Regi
 		chunkStore:  chunkStore,
 		flushQueues: make([]*util.PriorityQueue, cfg.ConcurrentFlushes),
 		wal:         &noopWAL{},
+		limits:      limits,
```

Comment:
Why do we need limits for flusher?

Reply:
Missing limits is what caused panic in the referenced issue. This PR is fixing that (limits were used by "Idle" series, this PR stops the loop that can generate them), but it also sets limits to avoid panics if flusher code accidentally tries to use them again.